com.strlen.TreeSheets.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.strlen.TreeSheets
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: treesheets
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  - /include
+  - /lib
+modules:
+  - name: TreeSheets
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DTREESHEETS_WITH_STATIC_WXWIDGETS=ON
+      - -DCMAKE_BUILD_TYPE=Release
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/aardappel/treesheets
+        commit: 5d97630e8f55199e8f9e28bd5cd30c04d1e60f7c
+      - type: git
+        url: https://github.com/wxWidgets/wxWidgets
```

Comment:
cmake warns about some missing libraries, not sure if you need them:

```
CMake Warning at lib/wxWidgets/build/cmake/init.cmake:342 (message):
  libsecret not found, wxSecretStore won't be available

CMake Warning at lib/wxWidgets/build/cmake/init.cmake:471 (message):
  webkit not found or enabled, wxWebview won't be available

CMake Warning at lib/wxWidgets/build/cmake/init.cmake:533 (message):
  Libnotify not found, it won't be used for notifications
```

Reply:
Thanks for your review! They are not needed indeed.